src/massive/munit/Assert.hx

```diff
@@ -226,4 +226,22 @@ class Assert
 	{
 		throw new AssertionException(msg, info);
 	}
+    
+    /**
+     * Assert that an expectation was thrown. Can expect strings and non-strings.
+     *
+     * @param   code                a function which should throw an exception 
+     * @return                      the exception that was thrown      
+     * @throws  AssertionException  if no expectation is thrown
+     */
+    public static function throws(code:Dynamic):Dynamic
```

Comment:
A few points
- You're not passing posinfos so when it fails the line of failure in the test wont be correct
- This seems to catch any exception and not a specific one you expect (like a specific string or class) so it's impossible to know if what was thrown was what you were expecting. i.e. the test may look lke its passing but shouldn't.
- The indentation doesn't match the rest of the file